test/box.bondingcurve.test.ts

```diff
@@ -25,7 +25,7 @@ describe('Bonding Curve', () => {
     }
   })
 
-  describe('Happy flow', () => {
+  describe.skip('Happy flow', () => {
```

Comment:
Should this be skipped?

Reply:
nope i shouldnt have pushed this i just commented them out locally since they take so long